test/sql/test_stream_load/R/test_prepared_timeout

```diff
@@ -0,0 +1,225 @@
+-- name: test_prepared_timeout @slow
+create database db_prepared_timeout_normal_${uuid0};
+-- result:
+-- !result
+use db_prepared_timeout_normal_${uuid0};
+-- result:
+-- !result
+CREATE TABLE `test_normal` (
+  `id` int
+) ENGINE=OLAP
+DUPLICATE KEY(`id`)
+DISTRIBUTED BY HASH(`id`) BUCKETS 10
+PROPERTIES (
+ "replication_num" = "1"
+);
+-- result:
+-- !result
+[UC]shell: curl --location-trusted -u root: -H "label:test_normal" -H "db:db_prepared_timeout_normal_${uuid0}" -H "table:test_normal" -XPOST ${url}/api/transaction/begin
+-- result:
+0
+{
+    "Status": "OK",
+    "Message": "",
+    "Label": "test_normal",
+    "TxnId": 239,
```

Comment:
will this txnId be always the same?

Reply:
it may different, so here use `[UC]` to ignore the result